Client/Frontend/Widgets/SnackBar.swift

```diff
@@ -29,15 +29,19 @@ class SnackButton: UIButton {
         }
     }
 
-    init(title: String, accessibilityIdentifier: String, callback: @escaping SnackBarCallback) {
+    init(title: String, accessibilityIdentifier: String, bold: Bool, callback: @escaping SnackBarCallback) {
```

Comment:
adding a default value here will simplify cases where bold is not needed. `bold: Bool = false`